hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -64,6 +68,14 @@
 public class HiveCatalog extends BaseMetastoreCatalog implements SupportsNamespaces, Configurable {
   public static final String LIST_ALL_TABLES = "list-all-tables";
   public static final String LIST_ALL_TABLES_DEFAULT = "false";
+  private static final DynMethods.UnboundMethod ALTER_TABLE = DynMethods.builder("alter_table")
```

Comment:
I did kinda wonder if we should extract this to a package utility method ? But this seems fine with me if you want to go this route. It's only used in 2 places so I think this is fine

Reply:
+1, Agree with you on this, I think we can move this to the existing [MetaStoreUtil](https://github.com/apache/iceberg/blob/master/hive-metastore/src/main/java/org/apache/iceberg/hive/MetastoreUtil.java) in the package WDYT ?